src/normal.cc

```diff
@@ -479,16 +490,21 @@ void command(Context& context, NormalParams params)
                 else if (not is_blank(cmdline[0]))
                     RegisterManager::instance()[':'].set(context, cmdline.str());
 
-                EnvVarMap env_vars = {
-                    { "count", to_string(params.count) },
-                    { "register", String{&params.reg, 1} }
-                };
                 CommandManager::instance().execute(
                     cmdline, context, { {}, env_vars });
             }
         });
 }
 
+void command(Context& context, NormalParams params)
+{
+    EnvVarMap env_vars = {
+        { "count", to_string(params.count) },
+        { "register", String{&params.reg, 1} }
+    };
+    command(context, env_vars);
```

Comment:
either write env_vars directly in the parameter, or std::move it.

Reply:
Used `std::move`